portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html

```diff
@@ -0,0 +1,43 @@
+<div mat-dialog-title>
+  <h4>Submit Application</h4>
+</div>
+
+<div mat-dialog-content class="content">
+  <p>
+    Your application will be submitted to the [L/FNG]. After submission, you will be auto-directed to complete the local
```

Comment:
I remember the [L/FNG] was just a design placeholder, and that it is supposed to be the name of the actual government. I would recommend confirming this with the design team

Reply:
Fixed!